pkgs/development/python-modules/ankipandas/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, pandas
+, colorlog
+, pytest
+, pytest-cov
+, randomfiletree
+}:
+
+buildPythonPackage rec {
+  pname = "ankipandas";
+  version = "0.3.15";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-31kkXytwUHeua6TBZMyfuvmQa9KNKcK/ielwDqNj37w=";
+  };
```

Comment:
IMO using fetching sources with `fetchFromGitHub` is better and more transparent way of fetching sources, even if it's a bit more verbose.